pulpcore/cli/common/context.py

```diff
@@ -169,13 +169,19 @@ class PulpEntityContext:
         "rpm": "rpm_rpm_repository_href",
     }
 
+    # Subclasses for nested entities can define the parameters for there parent scope here
+    @property
+    def scope(self) -> Dict[str, Any]:
```

Comment:
I just noticed this: would a `Dict[str, str]` make sense or is there a possibility that the value could be something other than a string?

Reply:
That's a good question. I can try making it `Dict[str, str]` and whenever mypy complains we can investigate why.

Reply:
Actually i just realized that a path parameter could also be an integer (like for repository versions). We don't nest anything beneath such an url yet, but it is a possibility. WDYT?

Reply:
Good point. `Dict[str, Any]` works then.